php/libraries/Image.class.inc

```diff
@@ -0,0 +1,256 @@
+<?php declare(strict_types=1);
+/**
+ * File describe the Image class
+ *
+ * PHP Version 7
+ *
+ *  @category Imaging
+ *  @package  Main
+ *  @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *  @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ *  @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS;
+
+/**
+ * This class defines a Image to allows one to type hint function's parameters
```

Comment:
```suggestion
 * This class defines a Image to allows one to type hint function's parameters.
```

Reply:
... defines **an** Image to **allow** one ...